guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -43,17 +43,17 @@ ifeval::["{context}" == "{project-context}"]
 --foreman-proxy-dns-managed true \
 --foreman-proxy-dns-interface __eth0__ \
 --foreman-proxy-dns-zone __example.com__ \
---foreman-proxy-dns-forwarders __172.17.13.1__ \
---foreman-proxy-dns-reverse __13.17.172.in-addr.arpa__ \
+--foreman-proxy-dns-forwarders __192.0.2.1__ \
```

Comment:
Forwarders are optional and not strictly needed. You are pointing here to the gateway and that may be confusing. We're getting very much off topic here which is why I usually separate changes like this into their own PRs.

IMHO we should not recommend forwarders by default. Then explain why they're needed. *if* you run a managed nameserver (bind), then it can be used to point to an upstream recursor which does all the heavy lifting. If you don't specify it, bind will reach out to the root servers and rely on its own cache. If you don't run a managed nameserver (for example because you use Infoblox or point to an existing external nameserver), it's ignored.

Reply:
This command is an example of a basic configuration that is shown in the Installing guide, which means that this can be the command that is run for the first time, so I will keep the `--scenario` option.
As for forwarders, as far as I understand, they are used in a configuration that uses an external server as a recursor. This sounds like something further than basic, so I will remove this option from the command.